install.sh

```diff
@@ -564,30 +570,255 @@ compare_versions() {
   return 0
 }
 
-get_linux_distro() {
-  declare distro
-  if [[ -f /etc/os-release ]]; then
-    . "/etc/os-release"
-    distro="$NAME"
-  elif type lsb_release 1>/dev/null 2>&1; then
-    distro="$(lsb_release -si)"
-  elif [[ -f "/etc/lsb-release" ]]; then
-    . "/etc/lsb-release"
-    distro="$DISTRIB_ID"
-  elif [[ -f "/etc/debian_version" ]]; then
-    # Older Debian/Ubuntu/etc.
-    distro="debian"
-  elif [[ -f "/etc/SuSe-release" ]]; then
-    # Older SuSE/etc.
-    distro="suse"
-  elif [[ -f "/etc/redhat-release" ]]; then
-    # Older Red Hat, CentOS, etc.
-    distro="redhat"
+prepare_environment_for_werf() {
+
+  local override_prepare_environment_for_werf="$1"
+
+  [[ $override_prepare_environment_for_werf == "no" ]] && return 0
+
+  [[ $override_prepare_environment_for_werf == "auto" ]] && prompt_yes_no_skip "Need to install packages for werf?" "yes" || return 0
+
+  if ! is_command_exists git; then
+    install_package git
+  else
+    already_installed git
+  fi
+
+  if ! is_command_exists curl; then
+    install_package curl
+  else
+    already_installed curl
+  fi
+
+  if ! is_command_exists gpg; then
+    install_package gpg
+  else
+    already_installed gpg
+    echo
+  fi
+
+}
+
+prepare_environment_for_buildah() {
+
+  local override_prepare_environment_for_buildah="$1"
+
+  [[ $override_prepare_environment_for_buildah == "no" ]] && return 0
+
+  [[ $override_prepare_environment_for_buildah == "auto" ]] && prompt_yes_no_skip "Need to install packages buildah and set kernel values?" "yes" || return 0
+
+  if ! is_command_exists buildah; then
+    install_package buildah
+  else
+    already_installed buildah
+  fi 
+  # Get the current user's name
+  CURRENT_USER=$(get_user)
+
+  # Create the path /home/<current user>/.local/share/containers if it doesn't exist
+  CONTAINER_PATH="/home/${CURRENT_USER}/.local/share/containers"
```

Comment:
I'm afraid this is not as simple. PATH to this directory I believe is calculated inside of Buildah. We need to somehow get it even before we can run Buildah. Which means we must repeat the logic for calculating this PATH inside of Buildah with all its quirks. And there probably quite a few, what about:
1. Non-standard $HOME (`/app`)
1. Non-standard [XDG variables](https://specifications.freedesktop.org/basedir-spec/latest/#basics)
1. Some other ways to change path to this directory, e. g. some parameters in Buildah config files.

This needs research.